pnpm-lock.yaml

```diff
@@ -9,11 +9,11 @@ overrides:
   semver-max: file:./npm/private/test/vendored/semver-max
   is-odd: file:./npm/private/test/vendored/is-odd
 
-packageExtensionsChecksum: 317d4b6ce1de48c4819a6fafc025ffb3
+packageExtensionsChecksum: sha256-wjVw+jaxrnffsda5QzoSE1Cv0vO+BjrFnGucmF47zPg=
```

Comment:
Notice the only change to the lockfile is switching hash types, so pnpm v10 interpreted the updated pnpm-workspace.yaml the same as pnpm9 interpretted the package.json `pnpm` field.